ui/mirage/factories/agent.js

```diff
@@ -5,19 +5,28 @@ import { DATACENTERS } from '../common';
 
 const UUIDS = provide(100, faker.random.uuid.bind(faker.random));
 const AGENT_STATUSES = ['alive', 'leaving', 'left', 'failed'];
-const AGENT_BUILDS = ['1.1.0-beta', '1.0.2-alpha+ent', ...provide(5, faker.system.semver)];
+const AGENT_BUILDS = [
+  '1.1.0-beta',
+  '1.0.2-alpha+ent',
+  ...provide(5, faker.system.semver),
+];
```

Comment:
Our pre-commit hook caught this file and did some prettifying. Lines 22-26 are the only practically changed ones.

Reply:
Yeah, its super annoying. Anytime I notice this I make sure to `git commit --no-verify`.

Its probably worth tackling some infrastructure stuff like this for developer quality of life.